packages/core/android/src/main/java/com/airbnb/android/react/lottie/OnAnimationFailureEvent.kt

```diff
@@ -0,0 +1,26 @@
+package com.airbnb.android.react.lottie
+
+import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.WritableMap
+import com.facebook.react.uimanager.events.Event
+
+class OnAnimationFailureEvent
+constructor(surfaceId: Int, viewId: Int, private val error: Throwable) :
+    Event<OnAnimationFinishEvent>(surfaceId, viewId) {
```

Comment:
shouldn't this be something like `Event< OnAnimationFailureEvent >`

Reply:
Done [bca4e69](https://github.com/lottie-react-native/lottie-react-native/pull/1068/commits/bca4e6985f8909832ec320d15222879e7e1edd0b).